Macros/ACS_summary_geo.sas

```diff
@@ -213,6 +140,13 @@
 
 	   FamMarriedCouple: FamMaleholder: FamFemaleholder: Nonfamlivingalone: Nonfamnotlivingalone:
 
+	   FamIncomeLT75k_: FamIncomeGT200k_:
+	   
+       NumRenterCostBurden: NumRentSevereCostBurden: NumOwnerCostBurden: NumOwnSevereCostBurden:
+       RentCostBurdenDenom: OwnerCostBurdenDenom:
+
+       NumRentCst: NumOwnCstBurden:

```

Comment:
Line 148 - repeated from line 130?

Reply:
@lhendey yes. I've deleted the duplicates.